waiter/src/waiter/core.clj

```diff
@@ -1547,6 +1548,7 @@
                                      auth/wrap-auth-bypass
                                      handler/wrap-https-redirect
                                      pr/wrap-maintenance-mode
+                                     (pr/wrap-unsupported-waiter-headers unsupported-headers)
```

Comment:
How do you feel about moving this to `descriptor/compute-descriptor` (after the `headers/split-headers` invocation)? I think that function should be responsible for disallowing certain headers.

Reply:
That's a good idea, I agree. I'm going to confirm it has the same behavior for websockets because the `compute-descriptor function` will have called by the `:default-websocket-handler-fn` instead of `:websocket-request-acceptor`

Reply:
Okay the error needs to surface in the `websocket-request-acceptor` handler which doesn't call `descriptor/compute-descriptor` in its tree of function calls. I had to add a error handler and include this logic in the `discover-service-parameters` function

Reply:
How does websocket request acceptor decide on the service ID?